ciao-controller/internal/datastore/datastore_test.go

```diff
@@ -30,9 +30,9 @@ import (
 
 	"github.com/ciao-project/ciao/ciao-controller/types"
 	"github.com/ciao-project/ciao/ciao-controller/utils"
-	"github.com/ciao-project/ciao/ciao-storage"
+	storage "github.com/ciao-project/ciao/ciao-storage"
```

Comment:
^^^ ?

Reply:
Goimports made the change but the change is correct.  You can't use the package id ciao-storage in Go code as this is not a valid Go identifier.  You need to use storage.  This change makes the effective package name explicit.

Of course this change, while correct, is not really related to moving the uuid package.  It's just a by product of running goimports on the file.  I could split it out into a separate commit.

Reply:
I'll revert the "storage" change for this and the sqlite3db_test.go file.